docs/about/faq.rst

```diff
@@ -20,15 +20,15 @@ Precision medicine refers to the customization of healthcare strategies to the i
 
 Who is CIViC designed for?
 --------------------------
-CIViC is intended to bring together a diverse group of individuals interested in the clinical application and advancement of cancer genomics including health professionals (oncologists, pathologists and other clinicians), researchers (cancer genome and data scientists), and patient advocates.
+CIViC is intended to bring together a diverse group of individuals interested in the clinical application and advancement of cancer genomics including health professionals (oncologists, pathologists and other clinicians), researchers (cancer genome and data scientists), and patient advocates. CIViC does not provide any medical or healthcare products, services or advice, and is not for medical emergencies or urgent situations. Information contained on this website is not a substitute for a doctor's medical judgment or advice. Patients should always discuss their specific, individual health concerns with a doctor or health care professional.
 
 What are the characteristics of an 'ideal' CIViC curator/editor?
 ----------------------------------------------------------------
-Here are some guidelines: CIViC curatiors should express a strong interest in clinical cancer genomics. Curators should have expertise in relevant academic fields including basic cancer biology, oncology, genomics, clinical informatics, etc and ideally are training for or have already obtained a medical degree (e.g. PhD or MD). Additionally, we expect curators to be open to sharing information with the community, be respectful of other contributors, professional, truthful, ethical, and unbiased. There is sometimes pressure in academia to present one's research findings in a clinically relevant context. When CIViC evidence statements are created, the curator should aspire to conduct an objective assessment of how well the data supports any assertion of clinical relevance. All aspects of the CIViC project are open to debate and amendment, including these guidelines.
+CIViC curatiors should express a strong interest in clinical cancer genomics. Curators should have expertise in relevant academic fields including basic cancer biology, oncology, genomics, clinical informatics, etc and ideally are training for or have already obtained a medical degree (e.g. PhD or MD). Additionally, we expect curators to be open to sharing information with the community, be respectful of other contributors, professional, truthful, ethical, and unbiased. There is sometimes pressure in academia to present one's research findings in a clinically relevant context. When CIViC evidence statements are created, the curator should conduct an objective assessment of how well the data supports any assertion of clinical relevance. All aspects of the CIViC project are open to debate and amendment, including these guidelines.
```

Comment:
Should we note that Editors are required to fill out COIs here?

Reply:
Sounds good.

Reply:
added additional commit to address this